lsp/core/aws-lsp-core/src/content/schemaProviderBuilder.ts

```diff
@@ -0,0 +1,34 @@
+import { URI } from 'vscode-uri'
+import { ContentRequestMiddleware, ContentRequestMiddlewareDelegate } from './contentRequestMiddleware'
+import { SchemaProvider } from './schemaProvider'
```

Comment:
I think we should move away from the 'schema' naming scheme since this logic ultimately applies to URIs as a whole.

Reply:
I think that is a good idea, we can encapsulate the "schema resolver" terminology that is used in the json/yaml servers. I will do a renaming pass as a separate PR.